com.unity.render-pipelines.universal/Runtime/ForwardRendererData.cs

```diff
@@ -7,197 +7,81 @@
 
 namespace UnityEngine.Rendering.Universal
 {
+    /// <summary>
+    /// Deprecated, kept for backward compatibility with existing ForwardRendererData asset files.
+    /// Use StandardRendererData instead.
+    /// </summary>
+    [System.Obsolete("ForwardRendererData has been deprecated. Use StandardRendererData instead (UnityUpgradable) -> StandardRendererData", true)]
     [Serializable, ReloadGroup, ExcludeFromPreset]
     [MovedFrom("UnityEngine.Rendering.LWRP")]
     public class ForwardRendererData : ScriptableRendererData
     {
-#if UNITY_EDITOR
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1812")]
-        internal class CreateForwardRendererAsset : EndNameEditAction
-        {
-            public override void Action(int instanceId, string pathName, string resourceFile)
-            {
-                var instance = CreateInstance<ForwardRendererData>();
-                AssetDatabase.CreateAsset(instance, pathName);
-                ResourceReloader.ReloadAllNullIn(instance, UniversalRenderPipelineAsset.packagePath);
-                Selection.activeObject = instance;
-            }
-        }
-
-        [MenuItem("Assets/Create/Rendering/Universal Render Pipeline/Forward Renderer", priority = CoreUtils.assetCreateMenuPriority2)]
-        static void CreateForwardRendererData()
-        {
-            ProjectWindowUtil.StartNameEditingIfProjectWindowExists(0, CreateInstance<CreateForwardRendererAsset>(), "CustomForwardRendererData.asset", null, null);
-        }
-#endif
-
-        [Serializable, ReloadGroup]
         public sealed class ShaderResources
```

Comment:
this is public API and need to be kept in place also.

Reply:
I put back all original labels.